app/views/templates/search/search_object.rb

```diff
@@ -629,41 +629,108 @@ def media_type(url)
-            license_public: true,
-            license_human: prefix + t('global.facet.reusability.open')
-          }
+         {
+            license_human: t('global.facet.reusability.open'),
+            license_name: t('global.facet.reusability.advanced-cc-by-nc-sa'),
+            license_CC_BY_NC_SA: true
+         }
+        elsif rights.index('http://creativecommons.org/licenses/by-sa') == 0
+         {
+            license_human: t('global.facet.reusability.open'),
+            license_name: t('global.facet.reusability.advanced-cc-by-sa'),
+            license_CC_BY_SA: true
+         }
+        elsif rights.index('http://www.europeana.eu/rights/out-of-copyright-non-commercial') == 0
+         {
+           license_human: t('global.facet.reusability.open'),
+           license_name: t('global.facet.reusability.advanced-out-of-copyright-non-commercial'),
+           license_OOC: true
+         }
         elsif rights.index('http://www.europeana.eu/rights/rr-f') == 0
-          {
-            license_public: false,
-            license_human: prefix + t('global.facet.reusability.permission')
-          }
+         {
+           license_human: t('global.facet.reusability.permission'),
+           license_name: t('global.facet.reusability.advanced-rrfa'),
+           license_RR_free: true
+         }
+        elsif rights.index('http://www.europeana.eu/rights/rr-p') == 0
+         {
+           license_human: t('global.facet.reusability.permission'),
+           license_name: t('global.facet.reusability.advanced-rrpa'),
+           license_RR_paid: true
+         }
+        elsif rights.index('http://www.europeana.eu/rights/rr-r/') == 0
+         {
+            license_human: t('global.facet.reusability.permission'),
+            license_name: t('global.facet.reusability.advanced-rrra'),
+            license_RR_restricted: true
+         }
         elsif rights.index('http://creativecommons.org/publicdomain/mark') == 0
-          {
+         {
             license_public: true,
-            license_human: prefix + t('global.facet.reusability.open')
-          }
+            license_name: t('global.facet.reusability.advanced-pdm'),
+            license_human: t('global.facet.reusability.open')
+         }
+        elsif rights.index('http://www.europeana.eu/rights/unknown') == 0
+         {
+            license_unknown: true,
+            license_name: t('global.facet.reusability.advanced-ucs'),
+            license_human: t('global.facet.reusability.open')
+         }
+        elsif rights.index('http://www.europeana.eu/rights/test-orphan') == 0
+         {
+            license_orphan: true,
+            license_name: t('global.facet.reusability.advanced-orphan-work'),
+            license_human: t('global.facet.reusability.open')
+         }
         else
-          {
-            license_public: true,
-            license_human: prefix + t('global.facet.reusability.open')
-          }
+         {
+            license_public: false,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.